### PR TITLE
Update to 0.11.7

### PR DESCRIPTION
**Pull request by Thimoteus**

### CODE REVIEW

File: trypurescript.cabal [hunk 1]
```diff
@@ -20,7 +20,7 @@ executable trypurescript
                    filepath -any,
                    Glob -any,
                    scotty -any,
-                   purescript ==0.11.6,
```
Comment (paf31) [inline trypurescript.cabal]: Could you please update the version of the package itself as well?

Comment (paf31) [reply trypurescript.cabal]: On line 2, I mean.

Comment (Thimoteus) [reply trypurescript.cabal]: To 0.11.6 as well?

Comment (paf31) [reply trypurescript.cabal]: Yes please

Comment (Thimoteus) [reply trypurescript.cabal]: Done!

**Comment by paf31** [comment]
Thanks!

**Pull request merged**